### PR TITLE
Feature/data hub addtl cols exploration

### PR DESCRIPTION
**Pull request by cheinz-sag**
Hi Tim and Marlene,

can you pls review?

@MWindel I'd propose to wait for Tim's feedback before you start reviewing as he'll check also for correctness of the content.

The PR is to be merged to:
- develop

This is doc for a not yet released CD feature. But I thought we can do the review in advance. Once the feature is released, I'll ping you and we can merge the branch (plus the associated change log branch). Then I'll also adapt the branch title and the PR title, which are currently non-standard.

Thanks
Christoph

### CODE REVIEW

File: content/datahub/working-with-datahub-bundle/configuring-offloading-jobs.md [hunk 1]
```diff
@@ -98,13 +98,11 @@ If you enter the additional result columns step for an active offloading pipelin
...
...supports you in this process with content completion and syntax highlighting. The **Change data type** controls helps you to define a function which changes the data type of the source definition. For example, the source definition is of type VARCHAR and corresponding values are always either true or false. Then you can select Boolean in the **Change data type** dropdown box to define a function which casts the VARCHAR values to BOOLEAN. Different target data types are available in the control, with some of them having options for dealing with non-matching values. For example, if you want to cast all values to type INTEGER and the non-matching literal N/A is processed, you can configure the casting function to use value 0 instead. If you have selected a data type you want to change to, click **Apply** to apply or **Cancel** to revert that type change. Note that functions you can apply to the source definition are not limited to the data type change functions provided under **Change data type**. In the source definition editor you can apply all SQL functions supported by Dremio, as listed under [SQL Function Categories](https://docs.dremio.com/software/sql-reference/sql-functions/).
```
Comment (MWindel) [inline content/datahub/working-with-datahub-bundle/configuring-offloading-jobs.md:101]:
```suggestion
When adding an additional result column, a dialog box for defining the column opens. You must define a unique column name as well as a source definition. Regarding the source definition, the first step is to specify a field from the base collection in the source definition editor. Then you can optionally apply SQL functions to adapt the data of this field to your needs, for example, by trimming whitespace or rounding decimal values. The source definition editor supports you in this process with content completion and syntax highlighting. The **Change data type** controls helps you to define a function which changes the data type of the source definition. For example, the source definition is of type VARCHAR and corresponding values are always either true or false. Then you can select Boolean in the **Change data type** dropdown box to define a function which casts the VARCHAR values to BOOLEAN. Different target data types are available in the control, with some of them having options for dealing with non-matching values. For example, if you want to cast all values to type INTEGER and the non-matching literal N/A is processed, you can configure the casting function to use value 0 instead. If you have selected a data type you want to change to, click **Apply** to apply or **Cancel** to revert that type change. Note that functions you can apply to the source definition are not limited to the data type change functions provided under **Change data type**. In the source definition editor you can apply all SQL functions supported by Dremio, as listed under [SQL Function Categories](https://docs.dremio.com/software/sql-reference/sql-functions/).
```

File: content/datahub/working-with-datahub-bundle/configuring-offloading-jobs.md [hunk 2]
```diff
@@ -98,13 +98,11 @@ If you enter the additional result columns step for an active offloading pipelin
...
...d preview its results by clicking **Load samples**. The system retrieves data of the associated collection, per default from the last 24 hours, and evaluates the source definition against that data. Results being **NULL** are filtered out. The maximum number of results is limited to 100. You can adjust the timeframe from which data is sampled using the time controls at the right top. The timeframe can cover at maximum the last seven days. To search for specific sample values, you can filter the current list of sample results with the filter controls at the top. The type of the sample results depends on the source data and the source definition. For complex types like **STRUCT** you can browse through the nested content of a sample entry by clicking at the nodes within the entry. If you want to set the source definition to a specific path of an entry, you can navigate to that path and click the hand icon right next to the path. You can also copy the path using the copy icon next to the path. Once you modify the source definition, the current sample results typically do not match anymore. Click **Reload** to retrieve a list of sample results with respect to the new source definition.
```
Comment (MWindel) [inline content/datahub/working-with-datahub-bundle/configuring-offloading-jobs.md:105]:
```suggestion
To validate the source definition and preview its results click **Load samples**. The system retrieves data of the associated collection, per default from the last 24 hours, and evaluates the source definition against that data. Results being **NULL** are filtered out. The maximum number of results is limited to 100. You can adjust the timeframe from which data is sampled using the time controls at the right top. The timeframe covers at maximum the last seven days. To search for specific sample values, filter the current list of sample results with the filter controls at the top. The type of the sample results depends on the source data and the source definition. For complex types like **STRUCT** browse through the nested content of a sample entry by clicking at the nodes within the entry. If you want to set the source definition to a specific path of an entry, navigate to that path and click the hand icon right next to the path. You can also copy the path using the copy icon next to the path. Once you modify the source definition, the current sample results typically do not match anymore. Click **Reload** to retrieve a list of sample results with respect to the new source definition.
```

File: content/datahub/working-with-datahub-bundle/configuring-offloading-jobs.md (hunk 1 above)
Comment (cheinz-sag) [reply content/datahub/working-with-datahub-bundle/configuring-offloading-jobs.md]: Done...

File: content/datahub/working-with-datahub-bundle/configuring-offloading-jobs.md (hunk 2 above)
Comment (cheinz-sag) [reply content/datahub/working-with-datahub-bundle/configuring-offloading-jobs.md]: Done...

**Review by MWindel — APPROVED** [review]

**Pull request merged**